src/swarm/neo/util/Batch.d

```diff
@@ -322,7 +322,7 @@ public scope class BatchReader ( Record ... )
 
         // Decompress into this.batch.
```

Comment:
I notice this comment is wrong.

Reply:
Maybe could you also add `Throws:` to the ctor doc? (`decompressSafe` can throw.)

Reply:
Updated.